share/spice/movie/movie.js

```diff
@@ -90,37 +81,99 @@
+                    image: null,
+                    url: item.links.alternate,
+                    is_retina: (DDG.is3x || DDG.is2x) ? "is_retina" : "no_retina"
+                };
+            }
+        }
+
+        function normalizeCast(item) {
+            return {
+                name: item.name,
+                character: item.characters && item.characters.join(", "),
+                heading: item.movie,
+                image: null,
+            }
+        }
+
+        function moviePicture(item) {
+            if (!item.alternate_ids || !item.alternate_ids.imdb) {
+                return;
+            }
 
-                $.ajaxSetup({ cache: true });
+            $.ajaxSetup({cache: true});
 
-                $.getJSON("/js/spice/movie_image/tt" + item.alternate_ids.imdb, function(data) {
-                    var path = data && data.movie_results && data.movie_results.length && data.movie_results[0].poster_path,
-                        image = path && "https://image.tmdb.org/t/p/w185" + path;
+            $.getJSON("/js/spice/movie_image/tt" + item.alternate_ids.imdb, function (data) {
+                var path = data && data.movie_results && data.movie_results.length && data.movie_results[0].poster_path,
+                    image = path && "https://image.tmdb.org/t/p/w185" + path;
 
-                    item.set({
-                        // fallback to lo-res:
-                        image: image || item.fallback_image,
+                item.set({
+                    // fallback to lo-res:
+                    image: image || item.fallback_image,
 
-                        // don't fallback in detail pane because
-                        // it looks silly with the tiny image:
-                        img: image,
-                        img_m: image
-                    });
+                    // don't fallback in detail pane because
+                    // it looks silly with the tiny image:
+                    img: image,
+                    img_m: image
                 });
+            });
+        }
+
+        function castPicture(item) {
+            $.ajaxSetup({cache: true});
+
+            $.getJSON("/js/spice/cast_image/" + item.name, function (data) {
+                var path = data && data.results && data.results.length && data.results[0].profile_path,
+                    image = path && "https://image.tmdb.org/t/p/w92/" + path;
+
+                item.set({
+                    // fallback to lo-res:
+                    image: image || item.fallback_image,
+
+                    // don't fallback in detail pane because
+                    // it looks silly with the tiny image:
+                    img: image,
+                    img_m: image
+                });
+            });
+        }
+
+        function getCasts(movies) {
+            var all_casts = [];
+            var length = movies.length;
+            for (var i = 0; i < length; i++) {
```

Comment:
https://encrypted.google.com/search?hl=en&q=cast%20of%20the%20avengers#hl=en&q=cast+of+lord+of+the+rings

Again only 1 movie, which makes the most sense. Otherwise you'd also have to deal with de-duplicating names and characters and it will get messy.